ts/Extensions/Boost/Boost.ts

```diff
@@ -366,7 +366,8 @@ export default Boost;
 
 /**
  * If set to true, the whole chart will be boosted if one of the series
- * crosses its threshold, and all the series can be boosted.
+ * crosses its threshold, and all the series can be boosted. If the
+ * `chart.zoomType` option is set, it will default to `false`.
```

Comment:
The doclet says `@default true`. If the description is right, this means the default should be `undefined`, which means an intelligent guess. The start of the description, "If set to true..." is also confusing, as it indicates that the default is not `true`.

Reply:
As the whole detection changed, I would argue that the default value is now still legit.